src/Serialization/main.jl

```diff
@@ -11,6 +19,8 @@ mutable struct SerializerState
     # io::IO
 end
 
+using RandomExtensions: check_interval
+using Pkg: upgrade_manifest
```

Comment:
We'll need some workaround here for julia 1.6. This is also the reason for the failing tests.